wetterdienst/download/download.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 import zipfile
 from io import BytesIO
-from multiprocessing import Pool
+from multiprocessing.dummy import Pool
```

Comment:
Shall we still use the "dummy" implementation or the real thing?

Reply:
After experiencing the issues with real multicore processing I'd trend to try to prevent using it. Another con in my point of view is that for downloading and unpacking of the zip (which is the only current usage) threading should be enough effort to speed things up.